packages/client/src/v2-events/features/workqueues/EventOverview/components/EventSummary.tsx

```diff
@@ -154,6 +160,18 @@ export function EventSummary({
     }
   })
 
+  const flagsArray = Array.isArray(event.flags)
+    ? (event.flags as string[]) // type assertion is needed since flags is not available in the FieldValue type
+    : []
+
+  const isCorrectionRequested = checkIfCorrectionRequested(flagsArray)
+  const eventWithUpdatedStatus = {
+    ...event,
+    'event.status': isCorrectionRequested
```

Comment:
We have discussed with Jon and decided that 'Correction requested' is not to be shown as a status on the event overview table for now, so this logic can be removed.